dali/operators/random/uniform_distribution_cpu.cc

```diff
@@ -30,10 +34,14 @@ generated.
     .AddOptionalArg("range",
       R"code(Range ``[min, max)`` of a continuous uniform distribution.
 
-This argument is mutually exclusive with ``values``.)code",
+This argument is mutually exclusive with ``values``.
+
+Warning: When specifying an integer type as ``dtype``, the generated numbers can go outside
```

Comment:
Should we use the `.. warning::` and `.. note::` boxes from rst?

Reply:
Done

Reply:
Shouldn't we clamp. I think this is the last thing I would expect TBH.

Reply:
There are one more Warning and Clamp.